src/controllers/softtakeover.h

```diff
@@ -1,56 +1,97 @@
 #pragma once
 
-#include <QHash>
+#include <chrono>
+#include <gsl/pointers>
+#include <unordered_map>
 
-#include "util/duration.h"
+#include "util/assert.h"
+#include "util/time.h"
 
 class ControlObject;
+class ControlPotmeter;
 
 class SoftTakeover {
   public:
-    // I would initialize it here but that's C++11 coolness. (Because it's a double.)
-    static const double kDefaultTakeoverThreshold;
+    // 3/128 units away from the current is enough to catch fast non-sequential moves
+    //  but not cause an audibly noticeable jump, determined experimentally with
+    //  slow-refresh controllers.
+    // TODO (XXX): Expose this to the controller mapping environment?
```

Comment:
https://github.com/mixxxdj/mixxx/blob/6e8f8c522ac2409e805ae761a20e02bf4ad468ca/res/controllers/Numark-NS6II-scripts.js#L235-L246

To be fair, a `SoftTakeveover::isIgnoring` would be a better solution here since it doesn't expose as much.

Reply:
Yes, do you have interest to add that?

Reply:
I'll see